### PR TITLE
sqlite integration added

### PR DESCRIPTION
**Pull request by erdogan-yesil**

### CODE REVIEW

File: thundra/integrations/modules/sqlite3.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        # raise appropriate error if api not supported (should reach the user)
+        self.__wrapped__.__enter__
+        return self
+
+
+class SqliteConnectionWrapper(wrapt.ObjectProxy):
+    db_name = None
+    host = None
+
+    def __init__(self, connection, db_name, host='localhost'):
```
Comment (serkan-ozal) [inline thundra/integrations/modules/sqlite3.py:34]: who pass a value for `host` arg?

Comment (serkan-ozal) [reply thundra/integrations/modules/sqlite3.py:34]: Is it possible to get it from `wrapper` args as you did for `db_name`?

File: thundra/integrations/sqlite3.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            if len(query) > 0:
+                operation = query.split()[0].strip("\"").lower()
+        except Exception:
+            pass
+
+        tags = {
+            constants.SpanTags['OPERATION_TYPE']: SQLiteIntegration._OPERATION_TO_TYPE.get(operation, ''),
+            constants.SpanTags['DB_INSTANCE']: connection.db_name,
+            constants.SpanTags['DB_HOST']: connection.host,
+            constants.SpanTags['DB_TYPE']: "sqlite",
```
Comment (serkan-ozal) [inline thundra/integrations/sqlite3.py:35]: You can use `CLASS_TYPE` constant defined above instead of using string literal here

Comment (erdogan-yesil) [reply thundra/integrations/sqlite3.py:35]: done

File: thundra/integrations/modules/sqlite3.py (hunk 1 above)
Comment (erdogan-yesil) [reply thundra/integrations/modules/sqlite3.py:34]: there is no "host" information in "args" or "kwargs" parameters.

**Review by serkan-ozal — APPROVED** [review]

**Pull request merged**